x-pack/plugins/ml/public/application/util/index_utils.ts

```diff
@@ -18,7 +18,25 @@ let indexPatternsContract: DataViewsContract | null = null;
 export async function loadIndexPatterns(indexPatterns: DataViewsContract) {
   indexPatternsContract = indexPatterns;
   const dataViewsContract = getDataViews();
-  indexPatternCache = await dataViewsContract.find('*', 10000);
+  const idsAndTitles = await dataViewsContract.getIdsWithTitle();
+
+  const dataViewsThatExist = (
+    await Promise.allSettled(
```

Comment:
The first time I visit the Data Frame Analytics page, I still see the error toasts - one for each index pattern for which the index doesn't exist.

<img width="1117" alt="image" src="https://user-images.githubusercontent.com/7405507/212299560-4db73402-5ebb-46f7-8ecb-bef6725ffe76.png">

Reply:
Fixed in https://github.com/elastic/kibana/pull/147916/commits/5086eff7b22517723ea0afba79d49b88364ce21c